src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
```

Comment:
Access modifier for the class NumberSupplier is missing. It's a good practice to define access modifiers explicitly. If this class is intended to be used within the package only, you can leave it as default; otherwise, consider adding public.